src/test/java/com/missafternoon/budjen/transactions/JSONInterpreterTest.java

```diff
@@ -0,0 +1,63 @@
+package com.missafternoon.budjen.transactions;
+
+import com.missafternoon.budjen.formatting.DateConverter;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class JSONInterpreterTest {
+
+    private JSONInterpreter target = new JSONInterpreter();
+    private TestTransactionData testData = new TestTransactionData();
+
+    @Test
+
+    public void writeJavaObjectIntoJSONString() {
+        String json = target.convertToString(testData.getTransactionData());
+        assertEquals("CREDIT, Let flat, £4200.00, 2018-06-08, DEBIT, Plastic Surgery, £25000.00, 2018-06-13", json);
+    }
+
+    @Test
+
+    public void readJSONStringIntoJavaObject() {
+        String string = "CREDIT, Work, £3000.00, 2018-11-14, DEBIT, Fuel, £25.00, 2018-12-16";
```

Comment:
Jackson throws an exception because this isn't JSON. Let's first define what we expect the JSON file to look like, then pass this format to the interpreter call.

Similarly, the assertion in the test above isn't using valid JSON.